src/__tests__/header.test.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable comma-dangle */
+/* eslint-disable quotes */
+import React from "react";
+import { BrowserRouter } from "react-router-dom";
+import TestRenderer from "react-test-renderer";
+import { Provider } from "react-redux";
+import store from "../Redux/configureStore";
+import Header from "../Components/Header";
+
+it("Test if header render correctly", () => {
+  const TREE = TestRenderer.create(
+    <Provider store={store}>
+      <BrowserRouter>
+        <Header />
+      </BrowserRouter>
+    </Provider>
+  );
+  expect(TREE).toMatchSnapshot();
+});
```

Comment:
- [ ] Good job with test coverage so, but as [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) for this project, " 
#### Testing requirements

- Create unit tests for pure functions in your Redux code.
- Create integration tests for your application using the React Testing Library.
- You may need to mock the access to the API, so that your tests don't send actual requests.
- You may need to mock the connection to the Redux Store. ", please kindly more unit tests for pure functions and more integration test.

Reply:
Thank you @Mihndim2020 , You review is very constructive and helps me alot